test/unit/spec/Util_spec.js

```diff
@@ -133,6 +135,41 @@ define(['util/Util', 'util/Logger'], function (Util, Logger) {
       });
     });
 
+    describe('postToUrl', function () {
+      beforeEach(function () {
+        spyOn($.fn, 'submit');
+      });
+
+      it('post the from when no query parameters', function () {
+        var container = $('<div/>');
+
+        Util.postToUrl('http://test.abc.com/', container);
+        expect(container.children().size()).toBe(1);
+
+        var form = $(container.children()[0]);
+        expect(form[0].outerHTML).toBe(
+          '<form method="POST" action="http://test.abc.com/" style="display:none;"></form>'
+        );
+        expect(form.submit.calls.count()).toBe(1);
+      });
+
+      it('post the from when has query parameters', function () {
```

Comment:
same typo here `form`... maybe this also could be `post the form with query parameters`